features/db.feature

```diff
@@ -61,6 +61,18 @@ Feature: Perform database operations
       Success: Exported
       """
 
+    When I run `wp db export /tmp/wp-cli-behat.sql --porcelain`
+    Then STDOUT should contain:
```

Comment:
Can you make this `The STDOUT should be:` to be explicit that there's no other output expected?